packages/smooth_app/lib/l10n/app_en.arb

```diff
@@ -4172,6 +4172,16 @@
   "@nutrition_facts_extract_failed": {
     "description": "Message to indicate that the extraction of nutrients from a picture failed"
   },
+  "attribute_preference_mandatory": "Mandatory",
+  "attribute_preference_very_important": "Very important",
+  "attribute_preference_important": "Important",
+  "attribute_preference_not_important": "Not important",
```

Comment:
In the Food preferences screen, `Mandatory, very important…` are values provided by the server.
Please do the same